### PR TITLE
again updated login page

### PR DESCRIPTION
**Pull request by azanbinzahid**

### CODE REVIEW

**Review by waahm7 — COMMENTED** [review]
46.8k lines of code seems too large for one PR and I am unable to determine how much it changes our codebase and how much code is for the template. Can you please try reducing it or maybe spliting it into multiple PR's?

**Comment by azanbinzahid** [comment]
All of the code is from the template.

File: src/main/webapp/js/navigation-loader.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ function addLoginOrLogoutLinkToNavigation() {
            navigationElement.appendChild(
               createListItem(createLink('/index.html', 'CS Opportunities Portal for Pakistani Students   ')));
            navigationElement.appendChild(
-              createListItem(createLink('/login', 'Login')));
+              createListItem(createLink('/login.html', 'Login')));
```
Comment (azanbinzahid) [inline src/main/webapp/js/navigation-loader.js:52]: redirected to html page instead of servlet

File: src/main/webapp/login.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Login V5</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (azanbinzahid) [inline src/main/webapp/login.html]: Added these two lines for css and navigation bar

File: src/main/webapp/login.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+					<span class="login100-form-title p-b-53">
+						Sign In With
+					</span>
+
+<!-- 					<a href="#" class="btn-face m-b-20">
+						<i class="fa fa-facebook-official"></i>
+						Facebook
+					</a>
+ -->
+					<a href="/login" class="btn-google m-b-20">
```
Comment (azanbinzahid) [inline src/main/webapp/login.html:53]: redirected this button to login servlet

File: src/main/webapp/login.html [hunk 4]
```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Login V5</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<script src="/js/navigation-loader.js"></script>
+    <link rel="stylesheet" href="/css/main.css">
```
Comment (azanbinzahid) [inline src/main/webapp/login.html:8]: Added these two lines for  css and navigation bar.

**Comment by azanbinzahid** [comment]
No major change in code base.

**Comment by azanbinzahid** [comment]
As per suggestions, I will make a better PR by incorporating all the changes and dependencies.

**Pull request closed without merging**